charts/camunda-platform/test/tasklist/deployment_test.go

```diff
@@ -247,16 +247,16 @@ func (s *deploymentTemplateTest) TestContainerSetContainerCommand() {
 	s.Require().Equal("printenv", containers[0].Command[0])
 }
 
-func (s *deploymentTemplateTest) TestContainerShouldSetTemplateEnvVars() {
```

Comment:
:thinking: The diff is a bit weird. It looks like you replace this test, was this intended?

Reply:
That's a good catch :grin: 
I will fix it now :bow: